src/components/TextInput/TextInput.test.tsx

```diff
@@ -109,7 +118,11 @@ describe("TextInput component", () => {
   describe("compontent snapshots", () => {
     it("default", () => {
       const { container } = render(
-        <TextInput label="Testing Input" fieldName="testing-input" />
+        <TextInput
+          id="test-input"
+          label="Testing Input"
+          fieldName="testing-input"
+        />
       );
       cleanAttributes(container, ["input-type-text"]);
```

Comment:
```suggestion
```
If we're not generating the ID here, this can come out. The function could be killed, too. If we do need to clean stuff again, it might not follow this same pattern.

Reply:
This is true for all snapshot tests in `TextArea` and `TextInput`